hack/install-router.sh

```diff
@@ -1,34 +1,71 @@
 #!/bin/bash
```

Comment:
@pweil- this has reached way over complexity - we need to figure out how you install routers in a saner way (admittedly, the bulk of this is secrets, but this is painful for end users too).  @csrwng this is the sort of thing that secrets has to solve for us.